docs/user/run.md

```diff
@@ -34,6 +34,7 @@ hostname (`-h $HOSTNAME`) and using the `-ip` Registrator option below.
 Option                           | Since | Description
 ------                           | ----- | -----------
 `-internal`                      |       | Use exposed ports instead of published ports
+`appendInternalPort`             |       | Append internal service port in services with only one port exposed
```

Comment:
I think it doesn't force append, because if you have metadata (without port) it has precedence over the auto-calculated name.